recipe/build.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
-CFLAGS="-fPIC" ./configure --shared --prefix=$PREFIX
+./configure --prefix=${PREFIX}  \
```

Comment:
This change (the removal of `"-fPIC"`) seems to be breaking Arrow builds on Travis-CI:
https://travis-ci.org/pitrou/arrow/jobs/385063783#L2321

Reply:
I thought that we had `-fPIC` set globally now :-/